src/AspNet.Security.OAuth.VkId/VkIdAuthenticationScopes.cs

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+namespace AspNet.Security.OAuth.VkId;
+
+/// <summary>
+/// List of <see href="https://id.vk.com/about/business/go/docs/vkid/latest/vk-id/connection/work-with-user-info/scopes"> scopes </see>.
+/// </summary>
+public static class VkIdAuthenticationScopes
+{
+    /// <summary>
+    /// Grants access to first and last name, gender, profile photo, date of birth.
```

Comment:
If we make it less specific, then if new fields are added to their API our documentation isn't wrong.

```suggestion
    /// Grants access to personal information.
```